src/TinyPubSub/Subscriptions/Subscription.cs

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
Hard to follow what has actually changed when all lines are replaced. Line ending mismatch?

Reply:
SubscriptionActions<T> added to handle the weak references, and all subscription files were moved to the /subscriptions/ folder

Reply:
Ah! Makes sense!